src/test/java/tsundere/task/DeadlineTest.java

```diff
@@ -0,0 +1,40 @@
+package tsundere.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void testDeadlineToString() {
+        Deadline deadline = new Deadline("test ", "Jan 01 2024");
+        assertEquals("[D][ ] test (by: Jan 01 2024)", deadline.toString());
+    }
+
+    @Test
+    public void testDeadlineToSaveString() {
+        Deadline deadline = new Deadline("test ", "Jan 01 2024");
+        assertEquals("D,0,test ,Jan 01 2024", deadline.toSaveString());
```

Comment:
Consider leaving the space after the comma: `assertEquals("D, 0, test, Jan 01 2024",`